### PR TITLE
fix: ensure source & destination verticals can be null

### PR DESCRIPTION
**Pull request by jatinsandilya**
@tonyxiao please let me know if there is a better fix for this.

### CODE REVIEW

File: kits/cdk/models.ts [hunk 1]
```diff
@@ -180,11 +180,11 @@ export const zRaw = {
       // TODO: Remove nullish now that pipelines are more fixed
       sourceId: zId('reso').optional(),
       sourceState: z.record(z.unknown()).optional(),
-      sourceVertical: z.string().optional(),
+      sourceVertical: z.string().optional().nullable(),
```
Comment (tonyxiao) [inline kits/cdk/models.ts:183]: `nullish` would do it, but yea same thing. This is right.

**Comment by tonyxiao** [comment]
looks like it is already merged though so will close it.

**Comment by jatinsandilya** [comment]
@tonyxiao 
This PR  needs to be merged in production not main. Would like this to be in prod, unable to test this in staging

**Comment by tonyxiao** [comment]
Merged main into prod so should fix this.

**Pull request closed without merging**